docs/index.rst

```diff
@@ -170,6 +170,7 @@ and lots of other useful information.**
 
    UsingPandasonRay/index
    UsingPandasonDask/index
+   UsingOmnisci/index
```

Comment:
We should also add something to this section.
https://modin.readthedocs.io/en/latest/developer/architecture.html#supported-execution-frameworks-and-memory-formats

Reply:
Thanks! Done